cocos/ui/view.ts

```diff
@@ -588,7 +588,8 @@ export class View extends Eventify(System) {
     }
 
     private _updateAdaptResult (width: number, height: number, windowId?: number) {
-        cclegacy.director.root.resize(width, height, windowId === undefined ? 1 : windowId);
+        // The default invalid windowId is 0
+        cclegacy.director.root.resize(width, height, (windowId === undefined || windowId === 0) ? 1 : windowId);
```

Comment:
I think the passing value should not be an invalid window id.

Reply:
Yes, it should be, just add the error handling here.